.github/workflows/test.yml

```diff
@@ -0,0 +1,48 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      max-parallel: 5
+      matrix:
+        python-version: ['3.6', '3.7', '3.8']
```

Comment:
Travis used to test 3.5 as well, should we include it here too?

(It is EOL so is a candidate for dropping.)

3.8 has been added, shall we add 3.9 as well?

Reply:
Good idea to add 3.9.